.eslintrc

```diff
@@ -124,7 +124,7 @@
         "use-isnan": 2,
         "valid-jsdoc": 0,
         "valid-typeof": 2,
-        "wrap-iife": 2,
+        "wrap-iife": 0,
```

Comment:
Maybe it's just me, but this feels like moving backwards. I think this rule is one of the important ones in keeping coding style consistent across repos.